apps/nextjs-website/src/lib/fetch/types/PageSection.ts

```diff
@@ -296,6 +296,17 @@ const ServiceCarouselSectionCodec = t.strict({
   cards: t.array(ServiceCardCodec),
 });
 
+const HighlightBoxSectionCodec = t.strict({
+  __component: t.literal('sections.highlight-box'),
+  eyelet: t.union([t.string, t.null]),
+  title: t.string,
+  body: t.string,
+  buttonText: t.union([t.string, t.null]),
+  buttonHref: t.union([t.string, t.null]),
```

Comment:
```suggestion
  link: t.union([LinkCodec, t.null]),
```
This is effectively a link. Use LinkCodec.

Feel free to change the variable's name if you think "link" isn't appropriate. I suggest keeping it that way so that its name already tells us the shape of the data inside